code/game/turfs/open.dm

```diff
@@ -331,10 +331,12 @@
 	for(var/mob/living/simple_animal/slime/M in src)
 		M.apply_water()
 
-	SEND_SIGNAL(src, COMSIG_COMPONENT_CLEAN_ACT, CLEAN_WEAK)
-	for(var/obj/effect/O in src)
-		if(is_cleanable(O))
-			qdel(O)
+	wash(CLEAN_WASH)
```

Comment:
I don't intend to be rude I just want our coders to be credited https://github.com/tgstation/tgstation/pull/52477